packages/gluestick/src/config/webpack/webpack.config.js

```diff
@@ -66,31 +65,44 @@ module.exports = (gluestickConfig: GSConfig): WebpackConfig => {
             {
               loader: 'babel-loader',
               options: {
-                plugins: ['transform-flow-strip-types'],
+                plugins: ['universal-import', 'transform-flow-strip-types'],
                 presets: ['es2015', 'react', 'stage-0'],
                 babelrc: false,
               },
             },
           ],
         },
         {
-          test: /\.(scss)$/,
-          use: [
-            'style-loader',
-            `css-loader?importLoaders=2${isProduction ? '' : '&sourceMap'}`,
-            'postcss-loader',
-            `sass-loader${isProduction
-              ? ''
-              : '?outputStyle=expanded&sourceMap=true&sourceMapContents=true'}`,
-          ],
-        },
-        {
-          test: /\.(css)$/,
-          use: [
-            'style-loader',
-            `css-loader?importLoaders=1${isProduction ? '' : '&sourceMap'}`,
-            'postcss-loader',
-          ],
+          test: /\.(s?css)$/,
```

Comment:
SCSS and CSS are combined because extract-css-chunks doesn't currently work with multiple loaders for CSS chunks (it's not impossible, but is looking for an owner to add that functionality).
This may impact build speed because of the partners CSS file. Measuring will be helpful.